components/identity/IdentityIndex.vue

```diff
@@ -80,11 +79,4 @@ watch(identity, () => {
     emit('change', identity.value)
   }
 })
-
-watch(
-  () => props.address,
-  (newAddress) => {
-    whichIdentity(newAddress)
-  }
-)
```

Comment:
I'm afraid that it would stop the update of identity if the address was changed after the component mounted.